docs/languages/powershell.md

```diff
@@ -0,0 +1,209 @@
+
+## PowerShell Extension multi-version support
+
+You can configure the PowerShell Extensions to use any version of PowerShell installed on your machine by following [these instructions](https://docs.microsoft.com/en-us/powershell/scripting/core-powershell/vscode/using-vscode?view=powershell-6#using-a-specific-installed-version-of-powershell)
+
+You can also change the version by clicking on the version number in  the lower right corner
+
+![Change PoswerShell Version](./images/powershell/pwshChangeVersion.png)
+
+From the [command pallette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette)
+enter
+```
+PowerShell: Show Session Menu
+
+```
+
+## Pester
+
+[Pester](https://github.com/pester/Pester/wiki/Pester)
+ is a Behavior-Driven Development (BDD) based unit test runner for PowerShell.
+Pester provides a framework for running unit tests to execute and validate PowerShell commands. Pester follows a file naming convention for naming tests to be discovered by pester at test time and a simple set of functions that expose a testing DSL for isolating, running, evaluating and reporting the results of PowerShell commands.
+
+Windows 10 and Server 2016 comes with Pester 3.40 pre-installed.
+to update Pester or to install the latest version on other platforms follow the [Pester installation instructions](https://github.com/pester/Pester#installation)
+
+![Pester CodeLens Integration](./images/powershell/pesterCodeLens.png)
+
+## Plaster
+
+ Plaster is a template-based file and project generator written in PowerShell. Its Purpose is to streamline the creation of PowerShell module projects, Pester tests, DSC configurations and more.
+ See the [Github Plaster repo](https://github.com/PowerShell/Plaster) for more information.
+
+ [Cmdlet Documentation](https://github.com/PowerShell/Plaster/blob/master/docs/en-US/Plaster.md)
+
+ Create a new Plaster Project from the [command pallette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette)
+( opened with <kbd>ctrl</kbd>-<kbd>shift</kbd>-<kbd>p</kbd>, <kbd>cmd</kbd>-<kbd>shift</kbd>-<kbd>p</kbd> on macOS) and typing Plaster:
+![Plaster Project](./images/powershell/cpPlasterCommand.png)
+
+
+## PSScriptAnalyzer
+
+The PowerShell Extension includes PSSCriptAnalyzer by default, and automatically performs analysis on PowerShell script files being edited in VSCode.
+
+[PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer#introduction) is a static code checker for Windows PowerShell modules and scripts. PSScriptAnalyzer checks the quality of Windows PowerShell code by running a set of rules. The rules are based on PowerShell best practices identified by the PowerShell Team and the community. Pester generates diagnostic rescords (errors and warnings) to inform users about potential code defects and suggests possible solutions for improvements.
```

Comment:
Can you remove the word "Windows" in "Windows PowerShell" since we have PowerShell Core as well?